test/index.test.ts

```diff
@@ -1,92 +1,104 @@
-import nock from 'nock'
-import approvemanApp from '../src'
-import { Probot } from 'probot'
-import prOpenedPayload from './fixtures/basic/pr.opened.json'
-import prReopenedPayload from './fixtures/basic/pr.reopened.json'
-import prSynchronizePayload from './fixtures/basic/pr.synchronize.json'
-import { setConfigToBasic, setConfigNotFound } from './utils/config'
-import { checkApproved, setSinglePreviousReview, verifyReviewDismissed } from './utils/review'
-const fs = require('fs')
-const path = require('path')
+import nock from "nock";
+import approvemanApp from "../src";
+import { Probot } from "probot";
+import prGeneric from "./fixtures/basic/pr.json";
+import { setConfigToBasic, setConfigNotFound } from "./utils/config";
+import { checkSuccessStatus } from "./utils/status";
+import {
+  checkApproved,
+  setSinglePreviousReview,
+  verifyReviewDismissed,
+} from "./utils/review";
+import fs from "fs";
+import path from "path";
 
-jest.setTimeout(30000)
+jest.setTimeout(30000);
 
-describe('Approveman tests', () => {
-  let probot: any
-  let mockCert: string
+const prOpenedPayload: any = prGeneric;
+prOpenedPayload.action = "opened";
+const prReopenedPayload: any = prGeneric;
+prOpenedPayload.action = "reopened";
+const prSynchronizePayload: any = prGeneric;
+prOpenedPayload.action = "synchronize";
+
+describe("Approveman tests", () => {
+  let probot: any;
+  let mockCert: string;
 
   beforeAll((done: Function) => {
-    fs.readFile(path.join(__dirname, 'fixtures/mock-cert.pem'), (err: Error, cert: string) => {
-      if (err) return done(err)
-      mockCert = cert
-      done()
-    })
-  })
+    fs.readFile(
+      path.join(__dirname, "fixtures/mock-cert.pem"),
+      (err: NodeJS.ErrnoException | null, cert: Buffer) => {
+        if (err) return done(err);
+        mockCert = cert.toString();
+        done();
+      },
+    );
+  });
 
   beforeEach(() => {
-    nock.disableNetConnect()
-    probot = new Probot({ id: 123, cert: mockCert })
-    probot.load(approvemanApp)
-  })
+    nock.disableNetConnect();
+    probot = new Probot({ id: 123, cert: mockCert });
+    probot.load(approvemanApp);
+  });
 
-  test('receive PR reopened', async (done) => {
-    setConfigNotFound()
-    checkApproved(done)
-    nock('https://api.github.com')
-      .get('/repos/tianhaoz95/approveman-test/pulls/1/files')
-      .reply(200, [
-        { filename: 'experimental/tianhaoz95/test.md' }
-      ])
-    await probot.receive({ name: 'pull_request', payload: prReopenedPayload })
-  })
+  test("receive PR reopened", async () => {
+    setConfigNotFound();
+    checkApproved();
+    checkSuccessStatus();
+    nock("https://api.github.com")
+      .get("/repos/tianhaoz95/approveman-test/pulls/1/files")
+      .reply(200, [{ filename: "experimental/tianhaoz95/test.md" }]);
```

Comment:
Codacy found an issue: ['magic numbers' are not allowed: 200](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6083797)